templates/tools/dockerfile/test/sanity/Dockerfile.template

```diff
@@ -31,7 +33,17 @@
   RUN python2 -m pip install simplejson mako virtualenv==16.7.9 lxml
   RUN python3 -m pip install simplejson mako virtualenv==16.7.9 lxml
 
-  <%include file="../../clang5.include"/>
+  # Install clang & clang-format
+  RUN apt-get install -y clang clang-format
+
+  # Install clang-tidy 6.0
+  # This is because clang-tidy 7.0 started treating compiler errors as tidy errors
+  # and there are a couple of files which are not properly compiled via tidy so it
+  # should be using 6.0 version until all compilation errors are addressed.
+  RUN apt-get install -y clang-tidy-6.0
```

Comment:
nit: in Dockerfiles, you always need to prepend apt-get install with `apt-get update && `